client/document/index.jsx

```diff
@@ -133,7 +136,14 @@ class Document extends React.Component {
 							>
 								<div className="masterbar" />
 								<div className="layout__content">
-									<WordPressLogo size={ 72 } className="wpcom-site__logo" />
+									{ 'jetpack-cloud' === sectionName ? (
```

Comment:
Ideally, this component should be section-agnostic. This isn't the case right now even before introducing this change, so I'm fine accepting this. I just wanted to mention that.